packages/xrpl/src/client/index.ts

```diff
@@ -668,6 +683,11 @@ class Client extends EventEmitter {
    */
   public getLedgerIndex = getLedgerIndex
 
+  /**
+   * @category Abstraction
+   */
+  public setNetworkIDAndBuildVersion = setNetworkIDAndBuildVersion
```

Comment:
Why does this need to be a public exported method?

Reply:
This is the part that I'm not happy with. If I make it as a helper function inside `autofill`, ESLint will complain about race condition `require-atomic-updates` when I set `client.networkID = ...` and `client.buildVersion=...` but I agree that this shouldn't be a public exported method. Any ideas?

Reply:
I made it a helper function for now (which I think is more reasonable), but it will have a bunch of eslint disable statements